### ISSUE TITLE
[Mac][Accessibility] Numeric Entries are not in the KeyViewLoop on focus change

### ISSUE THREAD

**Issue by netonjm** [xamarin/Xamarin.PropertyEditing#500]
Pressing tab between items, numeric entries are not accessible.

![focus_change_numeric](https://user-images.githubusercontent.com/1587480/51406512-a709fe00-1b59-11e9-9b85-5cdf14ae2f71.gif)

**Comment by ermau** [comment]
Cause: https://github.com/xamarin/Xamarin.PropertyEditing/commit/a4b2cb39a58f574a6d9e8a0cc7531e0d27d10dff

**Comment by CartBlanche** [comment]
What an odd bug. Other Numeric Controls do receive focus. Investigating.
